internal/backend/cron/jira.go

```diff
@@ -82,25 +82,25 @@ func (ct *Cron) checkJiraEventWatch(ctx context.Context, cid sdktypes.Connection
 	}
 
 	e := vs.GetValue(jira.WebhookExpiration)
-	twoWeeksFromNow := time.Now().UTC().AddDate(0, 0, 14)
 	t, err := time.Parse(time.RFC3339, e)
 	if err != nil {
-		l.Error("invalid Jira event watch expiration time during renewal check",
+		l.Warn("invalid Jira event watch expiration time during renewal check",
 			zap.String("expiration", e), zap.Error(err),
 		)
 		return false
 	}
 
 	// Update this event watch only if it's about to expire in less than 2 weeks.
+	twoWeeksFromNow := time.Now().UTC().AddDate(0, 0, 14)
```

Comment:
don't you want this configurable?

Reply:
no, I explicitly don't want users to be able to configure this on their own.

But even if we do add it in the future, it will be in an integration's config and not a cron config, and that's why a config struct isn't useful yet.

Reply:
not on their own, as ak config not user's